processing/format_qaqc_driver.py

```diff
@@ -0,0 +1,338 @@
+                                        log.write('Process terminated '
+                                                  'due to time out '
+                                                  'for upload_id: '
+                                                  f'{task.upload_id}')
+                                        if p.get('retry') < \
+                                                self.max_retries:
+                                            task = p.get('task')
+                                            p['process'] = \
+                                                pool.apply_async(
+                                                    upload_checks,
+                                                    (task.filename,
+                                                        task.upload_id,
+                                                        task.run_type,
+                                                        task.site_id,
+                                                        task.prior_process_id,
+                                                        task.zip_process_id,
+                                                        self.conn,
+                                                        self.is_test))
+                                            p['runtime'] = 0
+                                            retry = p.get('retry') + 1
+                                            p['retry'] = retry
+                                            log.write('Retry to run '
+                                                        'upload_id: '
+                                                        f'{task.upload_id}, '
+                                                        f'retry: {retry}')
+                                        else:
+                                            # terminate all process
+                                            # for this run
+                                            # and send email to team
+                                            if p.get('retry') > \
+                                                    self.max_retries:
+                                                (p
+                                                    .get('process')
+                                                    .terminate())
+                                                is_qaqc_successful = False
+                                    else:
+                                        p['runtime'] += self.time_sleep
+                            processes = s_processes
+                    # it will get here if all good, send out email to token
+                    if is_qaqc_successful:
+                        cmd = ('python '
+                               f'{self.email_gen_path} '
+                               f'{token}')
+                    else:
+                        cmd = ('python '
+                                f'{self.email_gen_team_path} '
```

Comment:
If we can make this a module -- great as well. Where is this code, btw? I'm not seeing it.